modules/nextflow/src/main/groovy/nextflow/Const.groovy

```diff
@@ -52,17 +52,17 @@ class Const {
     /**
      * The application version
      */
-    static public final String APP_VER = "19.12.0-edge"
+    static public final String APP_VER = "20.01.0-SNAPSHOT"
 
     /**
      * The app build time as linux/unix timestamp
      */
-    static public final long APP_TIMESTAMP = 1576839997768
+    static public final long APP_TIMESTAMP = 1578594512367
 
     /**
      * The app build number
      */
-    static public final int APP_BUILDNUM = 5244
+    static public final int APP_BUILDNUM = 5250
 
 
```

Comment:
Also these changes are generated automatically. Should not be committed.

Reply:
removed from commit by 
commit c8ce59e9d9ab182df7cd4077a255d3d50ba67f11